public/js/LoginAndSignup.js

```diff
@@ -99,67 +99,70 @@ function RegistrErrorAlert ()
 
 /*
 | register
-| create user information and url in to an array of object i.e @data 
+| create user information and url in to an array of object i.e @data
 | and make and ajax class to function ajaxCall() by sending @data and @functionName
 */
-function register () 
-{ 
+function register ()
+{
   var url       = "/signup";
   var email     = $('#email').val();
-  var token     = $('#token').val();   
+  var token     = $('#token').val();
   var username  = $('#username').val();
-  var password  = $('#password').val();   
-  
-  var data = 
+  var password  = $('#password').val();
+
+  var data =
     {
         url         : url,
-        parameter   : 
+        parameter   :
         {
           _token      : token,
           email       : email,
           username    : username,
-          password    : password   
+          password    : password
         }
     }
   checkItem(data);
-  var functionName =  arguments.callee.name;  
+  var functionName =  arguments.callee.name;
   ajaxCall( data, functionName );
 }
 
 
 /*
 | login
-| create user information and url in to an array of object i.e @data 
+| create user information and url in to an array of object i.e @data
 | and make and ajax class to function ajaxCall() by sending @data and @functionName
 */
-function login () 
+function login ()
 {
     var url       = '/login';
     var email     = $('#email').val();
-    var token     = $('#token').val();   
+    var token     = $('#token').val();
     var username  = $('#username').val();
-    var password  = $('#password').val();   
-    
-    var functionName =  arguments.callee.name;  
-    var data = 
+    var password  = $('#password').val();
+
+    var functionName =  arguments.callee.name;
+    var data =
     {
         url         : url,
-        parameter   : 
+        parameter   :
         {
           _token      : token,
           username    : username,
-          password    : password   
+          password    : password
         }
     }
     checkItem(data);
-    ajaxCall( data, functionName ); 
-} 
+    ajaxCall( data, functionName );
+}
 
-//console.log(window.location.hostname);      
-function checkItem (data) 
+//console.log(window.location.hostname);
+function checkItem (data)
 {
       if  ( data.parameter.email == '' || data.parameter.username == '' || data.parameter.password == '' )
       {
         swal("Oppss Login Failed", "Some required field not set!", "error")
       }
-}
\ No newline at end of file
+}
+
+
```

Comment:
Mixed double and single quotes.<br>Missing semicolon.<br>'alert' is not defined.